test/unit/AccountInfo.js

```diff
@@ -0,0 +1,90 @@
+import AccountInfo from "../src/account/AccountInfo.js";
+import AccountId from "../src/account/AccountId.js";
+import Hbar from "../src/Hbar.js";
+import { Key } from "@hashgraph/proto";
+import Timestamp from "../src/Timestamp.js";
+import Duration from "../src/Duration.js";
+import TokenRelationshipMap from "../src/account/TokenRelationshipMap.js";
+import Long from "long";
+
+/**
+ *  sendRecordThreshold and receiveRecordThreshold are deprecated and not included in this test nor toJSON
+ */
+
+describe("AccountInfo", function () {
+    it("should construct AccountInfo JSON object", function () {
+        const props = {};
+        props.accountId = new AccountId(1, 1, 1);
+        props.contractAccountId = "0000000000000000000000000000000000000001";
+        props.isDeleted = false;
+        props.proxyAccountId = "0000000000000000000000000000000000000002";
+        props.proxyReceived = new Hbar(1);
+        props.key = new Key();
```

Comment:
Two things:
1) That is the wrong `Key` type
2) Should use a more complex key type to show it working correctly.